CHANGELOG.rst

```diff
@@ -1,3 +1,7 @@
+**2.0.3 - 10/30/2024**
```

Comment:
I think this should be a minor version, so 2.1.0, since we're adding significant new functionality